### ISSUE TITLE
Create a benchmark based on PDEBench

### ISSUE THREAD

**Issue by Deathn0t** [deephyper/benchmark#11]
Let's define a benchmark suite based on the PDEBench: https://github.com/pdebench/PDEBench

**Comment by Deathn0t** [comment]
@iamyixuan advanced on this here: https://github.com/deephyper/benchmark/tree/main/lib/PINNBench/Diffusion-reaction

Next, we need to record learning curves.

**Comment by Deathn0t** [comment]
- [ ] document the possible values for `DEEPHYPER_BENCHMARK_DATASET`, what happens if an other dataset is used? it looks like the data generation is happening in the installation?
- [ ] the `flops` could be an additional objective in MOO
- [ ] would be great to have more parameters to tune such as:
  - regularization hyperparameters
    - [ ] `dropout_rate`
    - [ ] `early_stopping_patience`
    - [ ] `batch_norm` (batch normalization)
    - [ ] `weight_decay`
  - `skip_co` for skip connections (can be just a boolean with "regular skip co")
  - stochastic weight averaging
  - `optimizer`

**Comment by iamyixuan** [comment]
How to check the checkboxes above? Tried many times and only got to check two.

**Comment by Deathn0t** [comment]
- [ ] make weights as hyperparameters
